### PR TITLE
Habersaath: Cleaned config and added Local Mesh

### PR DESCRIPTION
**Pull request by FFHener**
Removed the old wireguardtunnel that was used in the beginning with a 60g link to the neighbour. In the meantime a new node appered in the neighbourhood. Trying to mesh with ffunk39 via existing antennas as an experiment

### CODE REVIEW

File: group_vars/location_habersaath/networks.yml [hunk 1]
```diff
@@ -88,7 +88,7 @@ location__channel_assignments_11a_standard__to_merge:
   habersaath-ap-c4: 48-20
   habersaath-ap-c5: 36-20
   habersaath-ap-c6: 40-20
-  habersaath-w-nf-5ghz: 44-20
+  habersaath-w-nf-5ghz: 34-20
```
Comment (spolack) [inline group_vars/location_habersaath/networks.yml:91]: 36 you mean?

**Comment by Akira25** [comment]
@FFHener @noxilixon Can this be merged soon?

**Comment by FFHener** [comment]
> @FFHener @noxilixon Can this be merged soon?

Yes, I converted it to an open PR.

**Review by Akira25 — APPROVED** [review]
Looks good to me.

**Pull request merged**